docs/general/ad-filtering/create-own-filters.md

```diff
@@ -392,6 +393,12 @@ Rules with `$domain` modifier as `regular_domain` or `any_tld_domain` are suppor
 
 :::
 
+:::caution Restrictions
+
+In AdGuard Browser Extension MV3 regexs and any tld domains are not supported.
```

Comment:
This is documentation, it must clearly indicate when we're using technical terms.

Something like this:
```suggestion
In AdGuard Browser Extension MV3 `regexp` and `any_tld_domain` are not supported.
```